File: C++/1. Two Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector <int> index;
        int size = nums.size();
        for (int i = 0; i<size; i++){
            int k = target -  nums[i];
            for (int j = i+1; j>i && j<size; j++){
                if (nums[j]==k){
                    index.push_back(i);
                    index.push_back(j);
                    break;
                }
            }
        }
        return index;
    }
};

File: C#/1. Two Sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int n = nums.Length;
        for(int i=0;i<n;i++){
            for(int j=i+1;j<n;j++){
                if(nums[j]==target-nums[i]){
                    return new int[]{i,j};
                }
            }
        }
        return new int[]{};
    }
}
